src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceEducation.vue

```diff
@@ -262,3 +272,8 @@ export default defineComponent({
   },
 });
 </script>
+<style>
```

Comment:
You should be able to use the color prop and remove `.error-message` class all together

https://vuetifyjs.com/en/api/v-alert/#props-color

Reply:
Ah sorry, this is for just displaying text in red? In that case I _think_ `text-error` would work

Reply:
we are using :rules to enforce validation on the checkboxes, but we do not want error messages to show for both of them individually(as per figma designs) so for now we added a div that gets toggled based on the :rules validation.
Note: need to revisit this an clean up the code to toggle validation message via :rules prop